### ISSUE TITLE
[Contact] Re: REDapp - Does not launch on Win 11 Oracle Jave or OpenJDK when clicking on the jar file

### ISSUE THREAD

**Issue by BadgerOnABike** [WISE-Developers/Project_issues#205]
### Message Body

Currently the release of REDapp does not launch via the jar by double clicking.  A user needs to activate the jar file from `cmd` with `java -jar REDapp.jar` to have it launch.

The users that would expect double clicking to work likely would not go down the path of trying to hit the utility via `cmd`.

**Comment by BadgerOnABike** [comment]
To assist in sorting out what may need to be done within the installer.
![image](https://github.com/user-attachments/assets/b44059df-6e61-4499-9abe-a1c1697b025b)
Or is the suggestion that the way the system is trying to talk to java isn't a proper call? I guess I'm asking if this solve is even in our control.

**Comment by spydmobile** [comment]
> To assist in sorting out what may need to be done within the installer. ![image](https://private-user-images.githubusercontent.com/10979751/390468400-b44059df-6e61-4499-9abe-a1c1697b025b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI3MjEyNjEsIm5iZiI6MTczMjcyMDk2MSwicGF0aCI6Ii8xMDk3OTc1MS8zOTA0Njg0MDAtYjQ0MDU5ZGYtNmU2MS00NDk5LTlhYmUtYTFjMTY5N2IwMjViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI3VDE1MjI0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMxZjFlOWJhM2ViNmQ5ODcxYTQ0NzI5N2Q5ODk4MzBhY2I5NzMwZmZhNWIyMWViYWU3YThmNmM2YzZlMDZlZDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.rNfq1psUAELu1UZ5x6CR7eYjc6ocShOz0qmrzbWIFPA) Or is the suggestion that the way the system is trying to talk to java isn't a proper call? I guess I'm asking if this solve is even in our control.

I think that that is coming from the default behaviour to use C:\Program Files\Yada yada, (note the spaces) which I am guessing it does not like and it may want the c:\Progra~1\Yada~1 the 8.3 safe path.

**Comment by spydmobile** [comment]
@BadgerOnABike that last bit should look like ```C:\Progra~1\Yada~1```

**Comment by BadgerOnABike** [comment]
This is coming from the redapp shortcut

**Comment by spydmobile** [comment]
yes, i think we blame the installer (making the shortcut) and perhpas use a better install script?
